microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
all .pngs: white-out the purple & gold + signs in top right

eg direct link to one of the .pngs (assuming "/images"/ dir):
https://raw.githubusercontent.com/MicrosoftDocs/edge-developer/user/pabrosse/json-viewer/microsoft-edge/devtools-guide-chromium/json-viewer/images/json-response.png